_posts/2023-09-22-licensing-update.md

```diff
@@ -0,0 +1,288 @@
+Title: An update on relicensing the MDAnalysis library (towards LGPLv2.1+)
+
+> ⚠️ **Disclaimer**
+> The MDAnalysis core team members are not
+> lawyers. As such the information provided here __does not, and is not
+> intended to constitute legal advice__. This blog post also does not
+> represent MDAnalysis' full legal position on software licensing; it
+> simply aims to inform MDAnalysis developers and users on certain
+> aspects of the MDAnalysis relicensing efforts and how we think it
+> may impact them going forward.
+>
+> Further information on open-source software licensing can be found
+> from sources such as the [Open Source Initiative][OSI],
+> [tl;drLegal][tldr legal] and the [Software Sustainability Institute][SSI licensing].
+>
+> Should you have any concerns about licensing, we always strongly
+> recommend getting legal advice before making any decisions on how
+> licensing changes may affect you.
+
+## Short description
+
+This blog post provides an update on MDAnalysis’ proposed license change. Specifically, we outline our
+decision to opt for a [GNU Lesser General Public License v2.1+ (LGPL v2.1+)][LGPL v2.1] instead of
+the [previously communicated]() [GNU Lesser General Public License v3+ (LGPL v3+)][LGPL v3].
+
+We also detail some [interim changes](#why-the-interim-gpl-v3-and-lgpl-v3-package-licenses-the-issue-with-apache-20)
+to the MDAnalysis [package license](#package-license), alongside some
+[upcoming actions for library contributors](#package-license).
+
+## Overview
+
+In November of 2022 the MDAnalysis team [announced a proposal to change the license of its core library](https://www.mdanalysis.org/2022/11/07/relicensing/#relicensing-mdanalysis)
```

Comment:
If you can use the Jekyll code that links to posts — something like post_url? That makes the site more robust